janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphOperationCountingTest.java

```diff
@@ -20,6 +20,8 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Iterables;
 import com.google.common.collect.Iterators;
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversal;
```

Comment:
Move tinkerpop dependencies to the line 52 (i.e. where all tinkerpop dependencies are listed)